client/src/components/LandingPage/UserReviewed.jsx

```diff
@@ -0,0 +1,126 @@
+import {
+	Card,
+	CardContent,
+	CardHeader,
+	Container,
+	Typography,
+} from '@mui/material';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import AccountCircle from '@mui/icons-material/AccountCircle';
+const userReviews = [
+	{
+		avatar: <AccountCircle></AccountCircle>,
+		name: 'Remy Sharp',
+		title: 'Book Title',
+		reviewTitle: 'Review Title',
+		review: "I absolutely love how versatile this product is! Whether I'm tackling work projects or indulging in my favorite hobbies, it seamlessly adapts to my changing needs. Its intuitive design has truly enhanced my daily routine, making tasks more efficient and enjoyable.",
+	},
+	{
+		avatar: <AccountCircle></AccountCircle>,
+		name: 'Travis Howard',
+		title: 'Book Title',
+		reviewTitle: 'Review Title',
+		review: "One of the standout features of this product is the exceptional customer support. In my experience, the team behind this product has been quick to respond and incredibly helpful. It's reassuring to know that they stand firmly behind their product.",
+	},
+	{
+		avatar: <AccountCircle></AccountCircle>,
+		name: 'Cindy Baker',
+		title: 'Book Title',
+		reviewTitle: 'Review Title',
+		review: 'The level of simplicity and user-friendliness in this product has significantly simplified my life. I appreciate the creators for delivering a solution that not only meets but exceeds user expectations.',
+	},
+];
+const UserReviewed = () => {
```

Comment:
Can this component be viewed anywhere? Nitpick: If it's a separate component I think it should have it's own folder, but if it's just data for the landing page we might call it userReviewedLandingPageData instead.

Reply:
I will move all components to separate folders! I thought since they would only be used on the LandingPage component it would be best to keep them in that folder.